### DISCUSSION TITLE
Is it correct to write dependencies in Gemfile?

### DISCUSSION THREAD

**Discussion by chiaraani** [ruby/rubygems#5758] [Q&A] [answered]
I have just created a gem with bundler. I chose these options `--test=rspec --linter=rubocop`.
And I found out that rake, rspec and rubocop where in the Gemfile instead of the gemspec. I wonder if that is correct.
```
source "https://rubygems.org"

# Specify your gem's dependencies in gemspec
gemspec

gem "rake", "~> 13.0"

gem "rspec", "~> 3.0"

gem "rubocop", "~> 1.21"
```

**Comment by simi** [comment, marked as answer]
That's correct. Your runtime dependencies (as mentioned in the comment) should be in gemspec. But for dev dependencies (tools useful for development) it is common to keep them in `Gemfile`. Alternatively you can specify them in gemspec using `add_development_dependency`.
